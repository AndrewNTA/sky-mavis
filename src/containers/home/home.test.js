import React from 'react';
import renderer from 'react-test-renderer';
import Home from './index';

jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn(),
}));
jest.mock('react-redux', () => {
  return {
    useDispatch: () => jest.fn(),
    useSelector: () => jest.fn(),
  };
});
test('Home should render home page', () => {
  const component = renderer.create(
    <Home />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});