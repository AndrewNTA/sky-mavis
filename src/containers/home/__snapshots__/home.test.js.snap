// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render home page 1`] = `
<div>
  <div
    className="h-header"
  >
    <div
      className="h-user-info"
    >
      <span
        className="h-dot-icon"
      />
      <span
        className="h-user-name"
      >
        Ronin Wallet
      </span>
    </div>
    <div
      className="h-user-account"
    >
      <img
        alt="account"
        className="h-account-icon"
        onClick={[Function]}
        src="account-icon.png"
        title="Logout"
      />
    </div>
  </div>
  <div
    className="h-main"
  >
    <div
      className="h-card-container"
    >
      <div
        className="h-card-top"
      >
        <div>
          <span
            className="h-card-wallet-title"
          >
            My wallet
          </span>
          <span
            className="h-card-number"
          >
            (7300 3700 3800 3334)
          </span>
        </div>
        <img
          alt="copy"
          className="h-copy-icon"
          src="copy-icon.png"
        />
      </div>
      <div
        className="h-card-bottom"
      >
        <div>
          <div
            className="h-currency-first"
          >
            ____ ___
          </div>
          <div
            className="h-currency-second"
          >
            ______ ___
          </div>
        </div>
        <img
          alt="logo"
          className="h-logo"
          src="ronin-logo-white.png"
        />
      </div>
    </div>
    <div
      className="h-control-group"
    >
      <div
        className="h-action-btn"
      >
        <div
          className="h-action-icon"
        >
          <img
            alt="card"
            className="h-card-icon"
            src="card-icon.png"
          />
        </div>
        <div
          className="h-action-text"
        >
          Deposit
        </div>
      </div>
      <div
        className="h-action-btn"
      >
        <div
          className="h-action-icon"
        >
          <img
            alt="send"
            className="h-send-icon"
            onClick={[Function]}
            src="send-icon.svg"
          />
        </div>
        <div
          className="h-action-text"
        >
          Send
        </div>
      </div>
      <div
        className="h-action-btn"
      >
        <div
          className="h-action-icon"
        >
          <img
            alt="swap"
            className="h-swap-icon"
            src="swap-icon.png"
          />
        </div>
        <div
          className="h-action-text"
        >
          Swap
        </div>
      </div>
    </div>
  </div>
  <div
    className="h-assets"
  >
    <div
      className="h-assets-title"
    >
      Assets
    </div>
    <div
      className="la-item"
    >
      <div
        className="la-icon"
      />
      <div>
        <div
          className="la-first-line"
        >
          ____ ___
        </div>
        <div
          className="la-second-line"
        >
          ______ ___
        </div>
      </div>
    </div>
    <div
      className="la-item"
    >
      <div
        className="la-icon"
      />
      <div>
        <div
          className="la-first-line"
        >
          ____ ___
        </div>
        <div
          className="la-second-line"
        >
          ______ ___
        </div>
      </div>
    </div>
  </div>
</div>
`;
